lib/catalog/martyrology.ts

```diff
@@ -3059,6 +3059,8 @@ export const Martyrology: { catalog: MartyrologyCatalog } = {
       dateOfDeath: 1917,
     },
     nativity_of_john_the_baptist: {
+      // Note: The title of Martyr is not indicated here, as Saint John the Baptist is not yet a Martyr at his birth.
```

Comment:
Just a nitpick.

/cc @TobiTenno, maybe this could be written more naturally / better.

```suggestion
      // Note: The title of a martyr is not indicated here, as Saint John the Baptist is not yet a martyr at his birth.
```

Reply:
probably, but i think it's poignant when referencing the title to have it specifically called out in this way.

I think `the title of Martyr` is the most precise way of saying it, as `a martyr` is not a title, you'd be in the state of being a martyr (the second case), but `the title of Martyr` is a universal, like a tag... so it's like saying `the tag of dependency` or `the dependency tag`, as opposed to "the tag of a dependency" sounds like you're speaking of a particular dependency.... sorry, way too much English nuance in this, but this is the most natural way I thought it could be done

Reply:
You’re right, @TobiTenno, there should be no indefinite artitle. :+1: Moreover, in English grammar, you describe different types of _phrases_ (e.g. noun phrase) as opposed to the Czech or Slovak grammar (where we deal more with individual words instead). That is a bit hard to understand and remember for me. As the phrase `title of martyr` is considered a single noun phrase (and not two nouns), there should be only a single (definite) article (and not two).

Just a final question regarding the phrase. I know that [titles](https://en.wikipedia.org/wiki/Title) (e.g. Mr/Mrs/Ms, PhD, Saint/Blessed, Martyr/Piest/Religious) are (sort of) part of name of a person in English. Now, as I see it, in this phrase, we are talking about the `martyr` title _in general_, but we also use the definite article before the phrase to denote that we are talking about this specific assignment of the title. **Do you still think we should use a capital initial letter in `Martyr`? Should we not lower-case it instead? :thinking:

Thank you very much! :pray:

Reply:
when speaking of Martyr in such a way that the definite article (`the`) would be used, a capital can be appropriate if you're speaking of the thing with particular importance. It's the reason names of things in English, like `English`, `Spanish`, are not lower-cased like they are in many Romance (Latin-derivative) languages.